traffic_ops/traffic_ops_golang/deliveryservicesregexes/deliveryservicesregexes.go

```diff
@@ -374,7 +382,7 @@ func Delete(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, fmt.Errorf("this create affected too many rows: %d", rowsAffected))
 		return
 	}
-	api.CreateChangeLogRawTx(api.ApiChange, fmt.Sprintf(`deleted deliveryservice_regex {"ds": %d, "regex": %d}`, dsID, regexID), inf.User, inf.Tx.Tx)
+	api.CreateChangeLogRawTx(api.ApiChange, "DS: "+string(dsName)+", ID: "+strconv.Itoa(dsID)+", ACTION: Deleted regular expression "+strconv.Itoa(regexID), inf.User, inf.Tx.Tx)
```

Comment:
can this be something like

Deleted a "+strconv.Itoa(dsr.Type)+" regular expression ("+dsr.Pattern+") in position "+strconv.Itoa(dsr.SetNumber

Reply:
I wanted to do that, but in this method I don't have any of that information and I don't know that it's all that easy to get it